build.gradle

```diff
@@ -15,20 +15,24 @@
  */
 
 buildscript {
+    ext.kotlin_version = '1.2.21'
```

Comment:
There is a newer version of kotlin; 1.2.30.

Reply:
I tried this for myself. You are correct in using 1.2.21 though because the Kotlin plugin version installed in Android Studio is 1.2.21 and not 1.2.30.